Coverity: Fix issues in udiskslinuxloop.c

```
Error: UNINIT (CWE-457): [#def40]
udisks-2.7.1/src/udiskslinuxloop.c:125: var_decl: Declaring variable "backing_file" without initializer.
udisks-2.7.1/src/udiskslinuxloop.c:160: uninit_use: Using uninitialized value "backing_file".
#  158|       }
#  159|   
#  160|->   if (backing_file == NULL)
#  161|       {
#  162|         udisks_loop_set_autoclear (UDISKS_LOOP (loop), FALSE);

Error: CLANG_WARNING: [#def41]
udisks-2.7.1/src/udiskslinuxloop.c:160:20: warning: The left operand of '==' is a garbage value
#  if (backing_file == NULL)
#      ~~~~~~~~~~~~ ^
udisks-2.7.1/src/udiskslinuxloop.c:125:3: note: 'backing_file' declared without an initial value
#  gchar *backing_file;
#  ^~~~~~~~~~~~~~~~~~~
udisks-2.7.1/src/udiskslinuxloop.c:132:7: note: Assuming the condition is false
#  if (g_str_has_prefix (g_udev_device_get_name (device->udev_device), "loop"))
#      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
udisks-2.7.1/src/udiskslinuxloop.c:132:3: note: Taking false branch
#  if (g_str_has_prefix (g_udev_device_get_name (device->udev_device), "loop"))
#  ^
udisks-2.7.1/src/udiskslinuxloop.c:160:20: note: The left operand of '==' is a garbage value
#  if (backing_file == NULL)
#      ~~~~~~~~~~~~ ^
#  158|       }
#  159|   
#  160|->   if (backing_file == NULL)
#  161|       {
#  162|         udisks_loop_set_autoclear (UDISKS_LOOP (loop), FALSE);

Error: COMPILER_WARNING: [#def42]
udisks-2.7.1/src/udiskslinuxloop.c: scope_hint: In function 'udisks_linux_loop_update'
udisks-2.7.1/src/udiskslinuxloop.c:169:7: warning: 'backing_file' may be used uninitialized in this function [-Wmaybe-uninitialized]
#       g_free (backing_file);
#       ^~~~~~~~~~~~~~~~~~~~~
#  167|                                    bd_loop_get_autoclear (g_udev_device_get_name (device->udev_device),
#  168|                                                           &error));
#  169|->       g_free (backing_file);
#  170|         if (error != NULL)
#  171|           {
```

--- a/src/udiskslinuxloop.c
+++ b/src/udiskslinuxloop.c
@@ -122,7 +122,7 @@ udisks_linux_loop_update (UDisksLinuxLoop        *loop,
   UDisksState *state;
   UDisksLinuxDevice *device;
   uid_t setup_by_uid;
-  gchar *backing_file;
+  gchar *backing_file = NULL;
   GError *error = NULL;
 
   daemon = udisks_linux_block_object_get_daemon (UDISKS_LINUX_BLOCK_OBJECT (object));

